virttest/qemu_storage.py

```diff
@@ -840,6 +841,9 @@ def create(self, params, ignore_errors=False):
                preallocated(optional)
                    if preallocation when create image, allowed values: off,
                    metadata. Default is "off"
+                compression_type(optional)
+                    if compression_type when create image, allowed values: zlib,
+                   zstd. Default is "zlib"
```

Comment:
```suggestion
                   zstd. 
```
The default `"zlib"` comes from the qemu layer, not the framework. To avoid misunderstanding, I suggest we should not expose any default product behaviors in the framework.
BTW, the previous similar docstring is not appropriate, like `preallocated`
Thanks.